symbiflow-xc7a50t/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -e
+set -x
+
+if [ x"$TRAVIS" = xtrue ]; then
+	CPU_COUNT=2
+fi
+
+wget -qO- https://storage.googleapis.com/symbiflow-arch-defs/artifacts/prod/foss-fpga-tools/symbiflow-arch-defs/continuous/install/15/20200608-212841/symbiflow-arch-defs-install-32defbb3.tar.xz | tar -xJ
```

Comment:
Why not extract the tar directly to `$SHARE_DIR/arch`

Reply:
I think that this requires work from the symbiflow arch defs standpoint. It would be better to have one tarball for each architecture there is, so that we can point to the files we need in each `symbiflow-<device>` conda package.
Currently the tar needs to be extracted and the xc7a50t device re-packed.